brutils/renavam.py

```diff
@@ -0,0 +1,47 @@
+def is_valid_renavam(renavam):  # type: (str) -> bool
```

Comment:
Ao invés de tipar os parâmetros e o retorno como comentário, utilize mesmo como se fosse uma tipagem forte em linguagens compiladas. 
```suggestion
def is_valid_renavam(renavam: str) -> bool:
```